examples/counter/www/index.html

```diff
@@ -15,25 +15,40 @@
 <script src="/jquery.min.js" type="text/javascript"></script>
 <script src="/api.js" type="text/javascript"></script>
 <script type="text/javascript">
+var onError = function(x){
+  return function(s){
+    return function(d){
+      return function(){
+        console.log(d); 
+      } 
+    }  
+  }
+}
+
+var updateCounter = function(response) {
+  return function(s){
+    return function(d){
+      return function(){
+        $('#counter').html('Counter: ' + response.value);
+      } 
+    }  
+  }
+};
+
 $(document).ready(function() {
   // we get the current value stored by the server when the page is loaded
-  getcounter(updateCounter, alert);
+  PS.App.getcounter(updateCounter)(onError)();
```

Comment:
This is a limitation of servant-jquery itself – the method name comes through entirely in lowercase. Might see about putting in a PR for this.